components/omega/OmegaBuild.cmake

```diff
@@ -19,11 +19,15 @@ set(OMEGA_DEFAULT_BUILD_TYPE      Release) # Debug or Release
 ###########################
 macro(setup_common_variables)
  
+  option(OMEGA_DEBUG "Turn on error message throwing (default OFF)." OFF)
+
   if(NOT DEFINED OMEGA_CXX_FLAGS )
     # initialize cxx flags as an empty list
     set(OMEGA_CXX_FLAGS "")
   endif()
 
+  set(OMEGA_LOG_DEFAULTPATH "${PROJECT_BINARY_DIR}/omega.log")
+
```

Comment:
I don't think this is what we want because I don't thin `${PROJECT_BINARY_DIR}` is the same as the current working directory, which is what I think we want.

Reply:
@xylar, in standalone build mode, PROJECT_BINARY_DIR is the same directory where the user invokes cmake command. In E3SM-build mode, PROJECT_BINARY_DIR is set by E3SM so the omega.log file is saved where E3SM binary is located. I thought this aligns with your suggestion as well as Phil's.

Reply:
@grnydawn

Yes, I get that but I don't think that's the behavior we want. If we build the model on one place and run in another, which is the normal case in Polaris, we want the log file to appear where you run, not where you build. We often build the model once and the run it dozens or even hundreds of times after that in different directories for different tests.  It's very important for following the model's progress and debugging problems that the log file appear in the run directory, not the build directory.

Does that make sense?

Reply:
@xylar , Thanks for the explanations. I think that they make sense to me now. I removed the macro and used just the base name of the file. So wherever it runs, the log file will be created in the current working directory.

Reply:
@grnydawn, perfect!